src/main/java/me/qoomon/maven/gitversioning/GitVersioningModelProcessor.java

```diff
@@ -125,7 +143,7 @@ private void init(Model projectModel) throws IOException {
             logger.info(extensionLogHeader(BuildProperties.projectGAV()));
         }
 
-        File pomFile = projectModel.getPomFile();
+        File pomFile = locatePom(projectModel.getProjectDirectory());
```

Comment:
I think we need to a comment why locatePom() is used instead of the more obvious projectModel.getPomFile()

Reply:
Should we store pomFile as a class variable to use it later again, instead of calling  locatePom again